tests/components/kodi/conftest.py

```diff
@@ -0,0 +1,111 @@
+"""The tests for Kodi binary sensor platform."""
+from unittest.mock import patch
+
+import pytest
+
+from homeassistant.components.kodi.const import (
+    CONF_WS_PORT,
+    DOMAIN,
+    WS_DPMS,
+    WS_SCREENSAVER,
+)
+from homeassistant.const import (
+    CONF_HOST,
+    CONF_NAME,
+    CONF_PASSWORD,
+    CONF_PORT,
+    CONF_SSL,
+    CONF_USERNAME,
+)
+from homeassistant.core import HomeAssistant
+
+from .util import MockConnection, MockWSConnection
+
+from tests.common import MockConfigEntry
+
+PATCH_KODI = "homeassistant.components.kodi"
+PATCH_KODI_CONNMAN = f"{PATCH_KODI}.connection_manager.KodiConnectionManager"
+
+TEST_KODI_CONFIG = {
+    CONF_NAME: "name",
+    CONF_HOST: "1.1.1.1",
+    CONF_PORT: 8080,
+    CONF_WS_PORT: 9090,
+    CONF_USERNAME: "user",
+    CONF_PASSWORD: "pass",
+    CONF_SSL: False,
+}
+
+
+def pytest_configure(config):
```

Comment:
I'm not familiar with this technique, so I'll let the core team opine, but I do think it's easily achievable with fixtures.

Reply:
Some other components also use custom marker (devolo_home_control, synology_dsm).

I'm really grateful for your time and help with my journey to Python and Home Assistant. If you are ever close to Frankfurt in Germany let me know, and I'll buy you at least a beer. 🍻